### PR TITLE
add doc for tidb_allow_tiflash_cop

### PR DESCRIPTION
**Pull request by guo-shaoge**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -874,6 +874,18 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+### `tidb_allow_tiflash_cop` <span class="version-mark">从 v7.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`OFF`
+- 控制是否允许使用 Cop/BatchCop 协议来计算下推到 TiFlash 的计算。Cop/BatchCop 是最初由 TiFlash 支持的协议。与 MPP 协议相比，性能没有差异，但更推荐使用 MPP 协议，因为 Cop/BatchCop 有可能在未来被弃用。
+   
+    * 0 或 OFF，意味着优化器不会针对 tiflash 使用 Cop/BatchCop。
```
Comment (joey-yez) [inline system-variables.md:886]:
```suggestion
    * 0 或 OFF，意味着优化器不会针对 TiFlash 使用 Cop/BatchCop。
```

File: system-variables.md [hunk 2]
```diff
@@ -874,6 +874,18 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
 - 这个变量用于控制当 [`replica-read`](#tidb_replica_read-从-v40-版本开始引入) 设置为 `closest-adaptive` 时，优先将读请求发送至 TiDB server 所在区域副本的阈值。当读请求预估的返回结果的大小超过此阈值时，TiDB 会将读请求优先发送至同一可用区的副本，否则会发送至 leader 副本。
 
+### `tidb_allow_tiflash_cop` <span class="version-mark">从 v7.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`OFF`
+- 当 TiDB 给 TiFlash 下发计算任务时，有三种方法（或协议）：Cop、BatchCop 和 MPP。相比于 Cop/BatchCop，MPP 协议更加成熟，提供更好的任务/资源管理。因此，更推荐使用 MPP 协议。
```
Comment (ran-huang) [inline system-variables.md:884]:
```suggestion
- 当 TiDB 给 TiFlash 下发计算任务时，有三种方法（或协议）可供选择：Cop、BatchCop 和 MPP。相比于 Cop 和 BatchCop，MPP 协议更加成熟，提供更好的任务和资源管理。因此，更推荐使用 MPP 协议。
```

**Review by ran-huang — APPROVED** [review]

**Review by joey-yez — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -874,6 +874,18 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
 - 这个变量用于控制当 [`replica-read`](#tidb_replica_read-从-v40-版本开始引入) 设置为 `closest-adaptive` 时，优先将读请求发送至 TiDB server 所在区域副本的阈值。当读请求预估的返回结果的大小超过此阈值时，TiDB 会将读请求优先发送至同一可用区的副本，否则会发送至 leader 副本。
 
+### `tidb_allow_tiflash_cop` <span class="version-mark">从 v7.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`OFF`
+- 当 TiDB 给 TiFlash 下发计算任务时，有三种方法（或协议）可供选择：Cop、BatchCop 和 MPP。相比于 Cop 和 BatchCop，MPP 协议更加成熟，提供更好的任务和资源管理。因此，更推荐使用 MPP 协议。
```
Comment (qiancai) [inline system-variables.md:884]:
```suggestion
- 当 TiDB 给 TiFlash 下推计算任务时，有三种方法（或协议）可供选择：Cop、BatchCop 和 MPP。相比于 Cop 和 BatchCop，MPP 协议更加成熟，提供更好的任务和资源管理。因此，更推荐使用 MPP 协议。
```

File: system-variables.md [hunk 4]
```diff
@@ -874,6 +874,18 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+
+    * 0 或 OFF：意味着优化器仅生成使用 TiFlash MPP 协议的计划。
+    * 1 或 ON：意味着优化器根据成本估算确定是否使用 Cop、BatchCop 和 MPP 协议。
```
Comment (qiancai) [inline system-variables.md:887]:
```suggestion
    * `0` 或 `OFF`：优化器仅生成使用 TiFlash MPP 协议的计划。
    * `1` 或 `ON`：优化器根据成本估算从 Cop、BatchCop 和 MPP 协议中选择一个用于生成执行计划。
```

File: releases/release-7.3.0.md [hunk 5]
```diff
@@ -172,6 +172,7 @@ v7.3.0 引入了以下主要功能。[功能详情](#功能详情)中列出的
 | 变量名 | 修改类型 | 描述 |
 |---|----|------|
 | [`tidb_opt_enable_mpp_shared_cte_execution`](/system-variables.md#tidb_opt_enable_mpp_shared_cte_execution-从-v720-版本开始引入) | 修改 | 该变量从 v7.3.0 开始生效，用于控制非递归的公共表表达式 (CTE) 是否可以在 TiFlash MPP 执行。 |
+| [`tidb_allow_tiflash_cop`](/system-variables.md#tidb_allow_tiflash_cop-从-v730-版本开始引入) | 新增 | 用于在 TiDB 给 TiFlash 下发计算任务时选择生成执行计划的协议。 |
```
Comment (qiancai) [inline releases/release-7.3.0.md:175]:
```suggestion
| [`tidb_allow_tiflash_cop`](/system-variables.md#tidb_allow_tiflash_cop-从-v730-版本开始引入) | 新增 | 用于在 TiDB 给 TiFlash 下推计算任务时选择生成执行计划的协议。 |
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**